Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update \
        ca-certificates \
        curl \
     && curl -s https://install.citusdata.com/community/deb.sh | bash \
-    && apt-get install -y postgresql-$PG_MAJOR-citus-9.4.=$CITUS_VERSION \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-9.5.=$CITUS_VERSION \
                           postgresql-$PG_MAJOR-hll=2.14.citus-1 \
                           postgresql-$PG_MAJOR-topn=2.3.0 \
     && apt-get purge -y --auto-remove curl \
```

Comment:
this breaks docker builds as we don't have pg13 packages for hll & topn yet

Reply:
see e9751947058235cd6dc014c17fd70bb2a9e26115

Reply:
Let's just remove these lines for now, until we have PG13 packages for hll and topn